src/pages/PublicProfile.vue

```diff
@@ -135,6 +142,24 @@ export default {
 
       await this.updateUser('locations', newLocations)
     },
+    async updateSocUser (fieldName, value, meta) {
+      var obj = {}
+      obj[fieldName] = value
+      await this.$store.dispatch('updateUser', {
+        userId: this.selectedUser.id,
+        attrs: {
+          [meta]: obj
+        }
+      })
+      this.notifySuccess('notification.saved')
+
+      // Hack to erase the content in locations AppSwitchableEditor (c.f. `computed.locationValue`)
```

Comment:
@Digital4design why is this needed? I would think this function would only be called when editing the social links.

Reply:
yes it would be call after saving the social link fields

Reply:
I removed this, was not needed.